### PR TITLE
Shcharbunou: lesson02

### PR DESCRIPTION
**Pull request by ddshcharbunou**
close: #38

### CODE REVIEW

File: shcharbunou/lesson02/maven-intro/src/test/java/by/it/shcharbunou/calculator/services/calculator_service/CalculatorServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    void testMultiply() {
+        // edge case
+        Assertions.assertEquals(0.010000000000000002, calculatorService.multiply(0.1, 0.1));
+        Assertions.assertEquals(100, calculatorService.multiply(50, 2));
+        // edge case
+        Assertions.assertEquals(0.05, calculatorService.multiply(0.1, 0.5));
+        // edge case
+        Assertions.assertEquals(0.0005, calculatorService.multiply(0.1, 0.005));
+    }
+}
```
Comment (rakovets) [inline shcharbunou/lesson02/maven-intro/src/test/java/by/it/shcharbunou/calculator/services/calculator_service/CalculatorServiceImplTest.java:77]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating newline"

File: shcharbunou/lesson02/maven-intro/src/test/java/by/it/shcharbunou/calculator/services/calculator_service/CalculatorServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class CalculatorServiceImplTest {
+
```
Comment (rakovets) [inline shcharbunou/lesson02/maven-intro/src/test/java/by/it/shcharbunou/calculator/services/calculator_service/CalculatorServiceImplTest.java:14]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson02/maven-intro/src/main/java/by/it/shcharbunou/calculator/utils/Operators.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package by.it.shcharbunou.calculator.utils;
+
+public enum Operators {
+
```
Comment (rakovets) [inline shcharbunou/lesson02/maven-intro/src/main/java/by/it/shcharbunou/calculator/utils/Operators.java:4]: "Violation: Useless blank lines
Solution: Remove blank lines"

**Review by rakovets — APPROVED** [review]
Approved with simple mistakes
